packages/office-ui-fabric-react/src/components/Image/Image.types.ts

```diff
@@ -100,7 +100,14 @@ export enum ImageFit {
    * Neither the image nor the frame are scaled. If their sizes do not match, the image will either be cropped or the
    * frame will have empty space.
    */
-  none = 3
+  none = 3,
+
+  /**
+   * The image will be centered horizontally and vertically within the frame and maintains its aspect ratio. It will
+   * behave as ImageFit.center if the height or width is less than the Image frame's height or width, but if image is
+   * fully larger than the frame it will behave as ImageFit.cover.
```

Comment:
fully larger isn't 100% clear without context. How about "if image is both taller and wider than the frame"

Reply:
updated